projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -95,11 +98,11 @@ module.exports = async function () {
 		'--source-maps'
 	);
 
-	if (fs.existsSync(path.join(CWD, 'webpack.config.js'))) {
-		webpack();
+	if (fs.existsSync(path.join(CWD, 'webpack.config.js')) || args.federation) {
```

Comment:
I don't think you need the `path.join` call here. You are in the current working directory already.

Reply:
:roll_eyes: :joy: 

`path.relative(CWD, path.resolve( path.join( CWD, 'webpack.config.js') ) )`